scene/property_utils.cpp

```diff
@@ -65,6 +65,17 @@ Variant PropertyUtils::get_property_default_value(const Object *p_object, const
 		*r_is_valid = false;
 	}
 
+	// Handle special case "Script" property
+	if (p_property == SNAME("script")) {
+		Ref<Script> ct_scr = p_object->get_custom_type_script();
+		if (ct_scr.is_valid()) {
+			if (r_is_valid) {
+				*r_is_valid = true;
+			}
```

Comment:
But if `r_is_valid` is true, no need to assign it again...

Reply:
`r_is_valid` is a pointer to a bool. It checks if the pointer isn't null, then dereferences it and sets it to true.

Reply:
Ah right...